3001-4000/LOC3170Gadama.xml

```diff
@@ -42,15 +42,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
+         <change who="SG" when="2020-05-19">transcription</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
             <place type="monastery">
-               <placeName>Gädamä Bäraqit</placeName>
-               <placeName type="alt">Gädamä Qaḥen</placeName>
-            </place>
+           
+                  <placeName xml:lang="gez" xml:id="n1">ገዳመ፡ በራቂት፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Gadāma Barāqit</placeName> 
+               <placeName xml:lang="gez" xml:id="n2" type="alt">ገዳመ፡ ቃኅን፡</placeName>
```

Comment:
ሕ/ḥ or ኅ/ḫ?

Reply:
By the way, if it is the church in Tigray, in Tigrigna it is also በራቒት.